src/user/models/user.entity.ts

```diff
@@ -45,12 +46,15 @@ export class UserEntity extends BaseEntitySchema {
   public role!: RoleType;
 
   @Prop({ type: String, default: null })
+  @Exclude()
```

Comment:
Remove all the attempts to serialize the entity including imports from this PR as we have a separate PR for that issue.

Reply:
Oh, I totally unnoticed this commit come along. 

Thanks for pointing it out